src/metrics/review-coverage.ts

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import gql from 'graphql-tag';
+
+import {getOrgRepos} from '../common-queries';
+import {GitHub} from '../gql';
+import {PullRequestCommitsQuery, PullRequestCommitsQueryVariables} from '../gql-types';
+import {RepoCommitsQuery, RepoCommitsQueryVariables} from '../gql-types';
+import {RepoPRsCommitsQuery, RepoPRsCommitsQueryVariables} from '../gql-types';
+
+import {getReviewsForPullRequest} from './review-latency';
+
+export class ReviewCoverageResult {
+  commits: Map<string, ReviewedCommit>;
+
+  constructor(commits: Map<string, ReviewedCommit>) {
+    this.commits = commits;
+  }
+
+  summary() {
+    const count = this.numReviewed();
+    return `There are ${this.commits.size} commits of which ${
+        count} are reviewed. \nReview coverage is ${
+        Math.round(count / this.commits.size * 100)}%.`;
+  }
+
+  numReviewed(): number {
+    let numReviewed = 0;
+    this.commits.forEach((x) => x.reviewed ? numReviewed++ : '');
+    return numReviewed;
+  }
+}
+
+type ReviewCoverageOpts = {
+  org: string,
+  repo?: string,
+  since?: string,  // ISO date string
```

Comment:
Should `since` be non-optional?

Reply:
I left is as optional since its a pain to pass in, and you may want for all time anyway. This param is only helpful if your repo is older than when reviews started, and you want better results there.